src/components/BingoBoardCard.vue

```diff
@@ -72,16 +78,36 @@ setInterval(() => {
 }, 200);
 
 function start() {
-  show.value = true;
+  // Logic for generating random numbers
+  startBtn.value = false;
+  showRandomNumber.value = true;
   startBingo.value = setInterval(() => {
-    bingo.value = ref(Math.floor(Math.random() * 99) + 1);
-    if (bingoNumbers.value.includes(bingo.value._value)) {
-      isBingo.value = true;
-    }
+    bingo.value = Math.floor(Math.random() * 99) + 1;
   }, 300);
 }
 
+watch(bingo, (newBingo) => {
+  if (
+    bingoBoardNumbers.value.includes(newBingo) &&
+    !includedColumnIndexes.value.includes(newBingo)
+  ) {
+    includedColumnIndexes.value.push(newBingo);

```

Comment:
set can be used here to avoid checking this everytime.

Reply:
Sir we will have to check everytime only because the numbers will ne randomly changing and we will have to check all the numbers.